### ISSUE TITLE
PIN THIS Info about NCP

### ISSUE THREAD

**Issue by ENDER1355** [NoCheatPlus/Issues#662]
NCP is no longer updated, if you want updated NCP go here 
https://github.com/Updated-NoCheatPlus

**Comment by MarkElf** [comment]
Yes, the repo's author that hasn't pushed an update or responded to any issues in years now will definitely get right to that.
